dynamic_rest/fields/fields.py

```diff
@@ -97,7 +99,9 @@ def __init__(
         self._serializer_class = serializer_class
         self.bound = False
         self.queryset = queryset
-        self.embed = embed
+        self.sideloading = sideloading
+        self.debug = debug
```

Comment:
Added this parameter to allow consumers to request debug mode to be turned on, resulting in extra response metadata being returned.